gulpfile.js/index.js

```diff
@@ -7,6 +7,7 @@ const uglify = require('gulp-uglify');
 const header = require('gulp-header');
 const concat = require('gulp-concat');
 const replace = require('gulp-replace');
+const cleanCSS = require('gulp-clean-css');
```

Comment:
FYI gulp-clean-css is using a pretty old clean-css version. You are better off using clean-css directly https://github.com/clean-css/clean-css#clean-css-for-gulp or https://github.com/clean-css/clean-css/issues/342

Reply:
I think it's ok for now. We don't really need any of the cool new clean-css features right now and using `gulp-clean-css` is just easier.